oss-internship-2020/gdal/README.md

```diff
@@ -0,0 +1,53 @@
+# GDAL Raster GeoTIFF Workflow
+
+```
+Build Tools: CMake/Ninja
+OS: Linux
+```
+
+### For testing: 
+`mkdir build && cd build`
```

Comment:
You should mention your project setup. Like, how to initialize the Git submodule and how to build the GDAL library.

You should also mention what other (Debian) packages need to be installed on the system.